sdk/internal/perf/recording_test.go

```diff
@@ -15,9 +15,11 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/internal/recording"
 )
 
+const proxyManualStartEnv = "PROXY_MANUAL_START"
```

Comment:
This will be removed fully after I refactor the rest of the tests to auto-start the proxy, and remove the proxy start step from the CI yaml.